### DISCUSSION TITLE
Trying to have zero padding around the chart

### DISCUSSION THREAD

**Discussion by danielchalmers** [apexcharts/apexcharts.js#4325] [Q&A] [answered]
You can see in this chart that there's unused space at the top, bottom, and left:

![image](https://github.com/apexcharts/apexcharts.js/assets/7112040/71e1371e-4b41-47d0-bc24-a09b1393771b)

With
- Chart.ParentHeightOffset = 0
- Chart.OffsetX = 15
- Grid.Padding.Top = -15

I was able to get the chart looking like this:

![image](https://github.com/apexcharts/apexcharts.js/assets/7112040/47da9376-867f-4834-a919-2ec34cbf4cc9)

But there is still a decent amount of extra padding on the right side of the chart (and bottom due to #4324); How can I remove this as well?

I really want zero margins on all sides. Is that easy to do with the current options?

**Comment by danielchalmers** [comment, marked as answer]
```
Grid = new Grid
{
    Padding = new Padding
    {
        Bottom = 0,
        Left = 0,
        Right = 0,
        Top = -15,
    },
},
Chart = new Chart
{
    Height = "100%",
    ParentHeightOffset = 0,
},
Xaxis = new XAxis
{
    Labels = new XAxisLabels
    {
        HideOverlappingLabels = true,
        Rotate = 0,
        Style = new AxisLabelStyle
        {
            FontSize = "9px",
        },
    },
},
Yaxis = new List<YAxis>
{
   new YAxis
   {
        Labels = new YAxisLabels
        {
            OffsetX = -15,
        },
   },
},
```

**Reply by Patzi275** [reply]
Please can you explain again ?
